packages/language-features/src/utils/augment-vls-ctx.ts

```diff
@@ -0,0 +1,27 @@
+import type { Code } from '@vue/language-core'
+
+export function augmentVlsCtx (content: Code[], getCodes: () => ` & ${string}`) {
```

Comment:
Does this still work if another plugin also augments the `__VLS_ctx` type in the same manner?